lib/mix/tasks/compile.thrift.ex

```diff
@@ -21,7 +21,7 @@ defmodule Mix.Tasks.Compile.Thrift do
   ## Command line options
 
     * `--force` - forces compilation regardless of modification times
-    * `--output` - set the output directory, overriding the `:thrift_output`
+    * `--out` - set the output directory, overriding the `:thrift_output`
```

Comment:
We could include the short option forms in the docs. Also the verbose option isn't mentioned.